Stratis.Bitcoin/Features/Consensus/TimeSyncBehavior.cs

```diff
@@ -444,7 +447,7 @@ private void AttachedNode_MessageReceived(Node node, IncomingMessage message)
                     VersionPayload version = node.PeerVersion;
                     if (version != null)
                     {
-                        TimeSpan timeOffset = this.dateTimeProvider.GetUtcNow() - version.Timestamp;
+                        TimeSpan timeOffset = version.Timestamp - this.dateTimeProvider.GetUtcNow();
```

Comment:
Ah right ha

Reply:
yeah :| writing code with time limit is not fun ;)